tests/unit/mathematics/linalg/operations/Eigen3_operations_unittest.cc

```diff
@@ -1466,6 +1468,56 @@ TYPED_TEST(LinalgBackendEigenAllTypesTest, SGMatrix_std_deviation)
 	    result.get_element(0), 2.581988897471611, get_epsilon<TypeParam>());
 }
 
+TYPED_TEST(LinalgBackendEigenAllTypesTest, SGMatrix_median)
+{
+	const index_t nrows = 3, ncols = 3;
+	SGMatrix<TypeParam> mat(nrows, ncols);
+	for (index_t i = 0; i < mat.size(); ++i)
+		mat[i] = i;
```

Comment:
I would generate a set of numbers with multiples to be a more "realistic" case, but I guess that is minor

Reply:
I just shuffled the values around a bit to make it more realistic, but I can add a multiple there.

Reply:
Probably fine